internal/api/ui/login/static/resources/scripts/change_password_check.js

```diff
@@ -1,16 +1,25 @@
 function CheckChangePwPolicy() {
-    let policyElement = document.getElementById("change-new-password");
-    let pwNew = policyElement.value;
-    let pwNewConfirmation = document.getElementById("change-password-confirmation").value;
+  let pwNew = document.getElementById("change-new-password");
+  let pwNewValue = pwNew.value;
+  let pwNewConfirmation = document.getElementById(
+    "change-password-confirmation"
+  );
+  let pwNewConfirmationValue = pwNewConfirmation.value;
 
-    if (ComplexityPolicyCheck(policyElement, pwNew, pwNewConfirmation) === false) {
-        policyElement.setAttribute("color", "warn");
-        return false;
-    } else {
-        policyElement.setAttribute("color", "primary");
-    }
+  if (ComplexityPolicyCheck(pwNew) === false) {
```

Comment:
is it normal to check bools in js this way? I'm used to check in this fashion: `if (! ComplexityPolicyCheck(...))`

Reply:
👍